functions/index.js

```diff
@@ -5,7 +5,11 @@ const queryString = require('query-string');
 const HONESTY_STORE_LOGO = 'https://honesty.store/assets/android/icon@MDPI.png';
 const BOT_USERNAME = 'honesty.store';
 
-exports.sendSlackMessage = functions.https.onCall(data => {
+exports.sendSlackMessage = functions.https.onCall((data, context) => {
+  if (context.auth.uid != 'EoBqPe3ZSEPovZvUMrEO7XTRdN42') {
```

Comment:
We don't want it to be an admin account. This way we can make it so that only the kiosk account has access without making it admin.

Reply:
It just means that for testing one account is being shared around which isn't ideal.

If this approach is the desired one then can we extract the uid to an environment variable and use `!==` instead of `!=`? (Ideally this should be a lint rule)